### ISSUE TITLE
Lnav is not building on aarch64-linux

### ISSUE THREAD

**Issue by Nebucatnetzer** [NixOS/nixpkgs#197512]
### Steps To Reproduce
1. Try to install lnav on a aarch64-linux machine `nix-shell -p lnav`

### Build log

https://hydra.nixos.org/build/194367636/nixlog/2

### Notify maintainers
@dochang @ma27

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-linux"`                                                                                                            
 - host os: `Linux 5.15.32, NixOS, 22.05 (Quokka), 22.05.20221018.47edaa3`                                                              
 - multi-user?: `yes`                                                                                                                   
 - sandbox: `yes`                                                                                                                       
 - version: `nix-env (Nix) 2.8.1`                                                                                                       
 - channels(root): `"nixos-22.05.2889.67e45078141"`                                                                                     
 - nixpkgs: `/nix/store/c2ashkynf2ihx7mkjwbsjcbd03wl1nd7-source`  
```

**Comment by ck3d** [comment]
Latest nixos (master branch) do not suffer from this issue on aarch64.

**Comment by Nebucatnetzer** [comment]
Does this mean that it will soon work in 22.05 as well?
Respectively 22.11?

**Comment by ck3d** [comment]
The maintainer did not back port the changes to 22.05. You could switch to nixos-unstable or in four weeks to nixos-22.11.

**Comment by Nebucatnetzer** [comment]
Ah yes good idea. I already have an unstable overlay for other packages and this package isn’t critically important so that would work as a workaround. I leave the issue open for the moment and will report again when 22.11 is out. On 4 Nov 2022, at 10:06, Christian Kögler ***@***.***> wrote:﻿
The maintainer did not back port the changes to 22.05. You could switch to nixos-unstable or in four weeks to nixos-22.11.

**Comment by Nebucatnetzer** [comment]
Resolved with 22.11
